src/server/common/simple_wml.hpp

```diff
@@ -91,7 +91,7 @@ class string_span
 	const char* begin() const { return str_; }
 	const char* end() const { return str_ + size_; }
 
-	int size() const { return size_; }
+	std::size_t size() const { return size_; }
```

Comment:
Did you check that this is safe?, when a code that calls these does something like 
```
int a = somefunction();
if (a < cfg.size()) {

}
```
Your patch changes behavior in a way that when somefunction() returned  -1 it was false before and not it might be true. So you probably have to check all codes that call size()

Reply:
That'd be `-Wsign-compare`, which doesn't occur as a result of that change.